### DISCUSSION TITLE
Pi Zero W performance with Genmon

### DISCUSSION THREAD

**Discussion by jtw1979** [jgyates/genmon#1081] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

Latest

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

How is the performance with the latest Genmon software on the Pi Zero W? (Not 2) Will I be satisfied with the performance or do I need a 2 W? It will only be used for Genmon. I'm running a Generac 8kW Nexus, will be using a Sandisk Ultra UH1 32GB card I have lying around and the generator is probably within 20ft of the router. I get full bars on wifi 6 with my phone out by the gen. It'll be running on the 2.4GHz band. Hoping to place the Pi inside the gen.

**Comment by jgyates** [comment]
As long as you are connecting the pi to your generator via direct serial link you should be fine. I would enable the "Optimize for slower CPU" setting on the Settings page. The pi zero should work well.
